solutions/agents/variables.tf

```diff
@@ -25,12 +25,12 @@ variable "cluster_resource_group_id" {
 }
 
 variable "cluster_config_endpoint_type" {
-  description = "Specify which type of endpoint to use for for cluster config access: 'default', 'private', 'vpe', 'link'. 'default' value will use the default endpoint of the cluster."
+  description = "Specify one of the following values to indicate which type of endpoint to use to access the cluster configuration: `default`, `private`, `vpe`, or `link`. The `default` value uses the default endpoint of the cluster."
```

Comment:
@acolleen To keep the descriptions short and sweet, I use some shortcuts. For values, I've been suggesting that we draw from the API reference precedent: `Possible values:`

> Specify the type of endpoint to use to access the cluster configuration. Possible values: `default`, `private`, `vpe`,`link`. The `default` value uses the default endpoint of the cluster.

Reply:
Thanks @SirSpidey I've made that update in the PR.